app/assets/javascripts/host_edit.js

```diff
@@ -442,7 +442,7 @@ function reload_host_params(){
 function reload_puppetclass_params(){
   var host_id = $("form").data('id');
   var url2 = $('#params-tab').data('url2');
-  var data = $("[data-submit='progress_bar']").serialize().replace('method=put', 'method=post');
+  var data = $("form").serialize().replace('method=put', 'method=post');
```

Comment:
@ohadlevy: the only places I was able to find that still use [data-submit='progress_bar'] are for host edit and those are not broken, because [host](https://github.com/theforeman/foreman/blob/develop/app/views/hosts/_form.html.erb#L8) and [hostgroup](https://github.com/theforeman/foreman/blob/develop/app/views/hostgroups/_form.html.erb#L2) _form are not completely consistent (there is a `:submit => "progress_bar"` in hosts form). Changing the hostgroup form to make it consistent will probably be the best. But there is a failed call to server in hosts form, I noticed just now, it has no obvious impact on UI, I will fix it as well.